easy_phi/static/js/main.js

```diff
@@ -234,7 +216,8 @@ var ep = window['ep'] || {
         }
     },
 
-    _parseWSMessage: function (message) {
+    _parseWSMessage: function (event) {
+        var message = event.data;
```

Comment:
this line allows to get rid of anonymous function in handler assignment. It is not critical from functional point of view